2_Last Position & Math Table.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8

```

Comment:
what is the `last position` in the title of the file and PR?

Reply:
It was the name of the test problem, so i used the same name for title.